tests/v2/validation/provisioning/rke2/etcd_backup_restore.go

```diff
@@ -32,14 +32,26 @@ const (
 	etcdnodeCount                = 3
 )
 
-func createSnapshot(client *rancher.Client, clustername string, generation int, namespace string) error {
+func CreateSnapshot(client *rancher.Client, clustername string, namespace string) error {
```

Comment:
Instead of making this function public it should be an extension.

Reply:
done